app/presenters/admin_auction_status_presenter/available.rb

```diff
@@ -4,7 +4,26 @@ def header
   end
 
   def body
-    I18n.t('statuses.bid_status_presenter.available.admin.body', end_date: end_date)
+    if total_bids > 0
```

Comment:
Yeah I think this is fine to leave as-is for now and then we can think about moving our crazy factory conditional into smaller parts and at the point we might want to surface this logic outside of this presenter.